### ISSUE TITLE
Binary rocks for multiple Lua versions

### ISSUE THREAD

**Issue by moteus** [luarocks/luarocks#907]
Does LuaRocks 3 suppose to support it?

**Comment by daurnimator** [comment]
What do you mean? One binary rock that contains code compiled for 3 different lua versions?
I can't imagine such a shared object to be easy to create.

**Comment by moteus** [comment]
I mean upload multiple rocks for each Lua version.
I make binary rocks and theirs names are same for any Lua version.

**Comment by Tieske** [comment]
good point. The Lua version dependency is only in the rockspec contents, not in its name.

**Comment by daurnimator** [comment]
Related to #788?

**Comment by moteus** [comment]
Maybe its possible create several rockspec files for the same version but for different Lua and then build rocks for them?

**Comment by moteus** [comment]
Maybe its possible create several rockspec files for the same version but for different Lua and then build rocks for them?

**Comment by Tieske** [comment]
But then the binary rocks would still have the same name
